index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Twitter</title>
+</head>
+<body>
+  <header class="header">
+    <img src="assets/icons/logo-twitter.svg" alt="Twitter logo" class="filter-red">
+    <h1>Home</h1>
+    <div class="search-bar">
+      <img src="assets/icons/search-outline.svg" alt="Search" class="filter-grey">
+      <p>Search</p>
+    </div>
+  </header>
+  <section class="left-bar">
+    <div class="button">
```

Comment:
You could have added a rounded gray background to the elements when hovering, like it is with Twitter currently. Your texts are getting bold when hovering though so it may have been a design choice.

Reply:
That was a thought, but I couldn't get it to work properly, looked like you saw straight through it though.